code/control/DeployPlanDispatcher.php

```diff
@@ -290,6 +284,10 @@ protected function getGitPrevDeploys($project) {
 	 * @return SS_HTTPResponse
 	 */
 	protected function getAPIResponse($output, $statusCode) {
```

Comment:
Probably will have to, since I think my next step is to break this up into dispatchers for 
- git stuff
- approval
- deploy planning
- deploy